_articles/ssl.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: article
+title: Setting Up SSL in Envoy
+---
+
+[//]: # ( Copyright 2018 Turbine Labs, Inc.                                   )
+[//]: # ( you may not use this file except in compliance with the License.    )
+[//]: # ( You may obtain a copy of the License at                             )
+[//]: # (                                                                     )
+[//]: # (     http://www.apache.org/licenses/LICENSE-2.0                      )
+[//]: # (                                                                     )
+[//]: # ( Unless required by applicable law or agreed to in writing, software )
+[//]: # ( distributed under the License is distributed on an "AS IS" BASIS,   )
+[//]: # ( WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or     )
+[//]: # ( implied. See the License for the specific language governing        )
+[//]: # ( permissions and limitations under the License.                      )
+
+[//]: # (Setting Up SSL in Envoy)
```

Comment:
should have caught this in the docs, but should this be "with" rather than "in"?